packages/cli/commands/project/create.js

```diff
@@ -61,6 +62,10 @@ exports.builder = yargs => {
       describe: i18n(`${i18nKey}.options.template.describe`),
       type: 'string',
     },
+    customRepoPath: {
```

Comment:
Small nit, but IMO "custom" is a little redundant here. Would just `repoPath` work?

Reply:
I was debating that, I can definitely make this change.

Reply:
I came here to just ask the same question as @brandenrodgers . :-) 
I think `repoPath` is short, and it's understood that it's going to be custom if you are using this flag.

Reply:
I also wonder if just `--path` is okay (to allow for non-GitHub sources). But that may be me overthinking it.

Reply:
Right now, we're only set up to allow downloads from GitHub repositories (we're using the GH API), so I think it makes sense to keep as `repoPath`. Thanks for the feedback!